pdp/auth.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+from webob.request import Request
+
+from pdp import wrap_auth
+from pdp.dispatch import PathDispatcher
+
+def user_login(environ, start_response):
+    '''
+    A WSGI app to register a verified email in the session and log in
+    '''
+
+    request = Request(environ)
+    session = environ.get('beaker.session', {})
+
+    if request.method == 'POST':
+        email = request.POST.get('email', None)
+        if not email:
+            start_response('400 Bad Request', [('Content-type', 'text/html; charset=utf-8')])
+            return ['Email must be included in POST body']
+        session['email'] = email
+        start_response('200 OK', [('Content-type', 'text/html; charset=utf-8')])
+        return json.dumps({'session_id': session.id})
+
+    if request.method == 'GET':
```

Comment:
There are other available HTTP verbs that this method doesn't support, so probably better to just say

```python
else:
    start_response('400 BAD REQUEST', [('Content-type', 'text/html; charset=utf-8')])
    return ['Login handler does not support {} requests'.format(request.method)]
```

or `"Login handler only supports POST requests"`